### ISSUE TITLE
Overview of all shares that were sent

### ISSUE THREAD

**Issue by audax** [Qabel/qabel-android#31]

**Comment by mr-gosh** [comment]
![filebrowser-granted-shares](https://cloud.githubusercontent.com/assets/3028703/14892530/827f8cfc-0d6b-11e6-9f9d-32c9e406a441.png)

**Comment by thechauffeur** [comment]
Since we cannot simply implement the removal of a share on a per contact basis (because technical backend reasons) I think this issue should be split.

On overview is very helpful and should be implemented here but the mockup also suggests that the implementation also give more functionality than it actually should. The mockup should only show a dialog for "remove all shares" and not a removal on a per contact basis.

**Comment by thechauffeur** [comment]
I guess no new mockup is needed because there already is a "remove all" button implemented.

tl;dr The functionality to remove shares on a per contact basis must be a new issue which is not planned for the "RC" milestone.
